static/js/impala/stats/manager.js

```diff
@@ -8,6 +8,10 @@ z.StatsManager = (function() {
     var STATS_VERSION = '2011-12-12';
     var PRECISION = 2;
 
+    $.ajax('/static/js/currency.json').then(function(c) {
```

Comment:
We usually set `data-static-url` to `<body>` so that you can use that as a base for requesting static files.